tests/unit/cirrus/build_tests.py

```diff
@@ -101,6 +111,43 @@ def test_execute_build_default_pypi(self):
             mock.call('. ./venv/bin/activate && python setup.py develop')
         ])
 
+    def test_execute_build_pypirc(self):
+        """test execute_build with pypirc provided settings"""
+        opts = mock.Mock()
+        opts.clean = False
+        opts.upgrade = False
+        opts.extras = []
+        opts.nosetupdevelop = False
```

Comment:
Same PropertyMock question...better yet check out the ctor notes here -- they can be kwargs http://www.voidspace.org.uk/python/mock/mock.html#mock.Mock.configure_mock